app/forms/state_file/ny_permanent_address_form.rb

```diff
@@ -12,6 +12,15 @@ class NyPermanentAddressForm < QuestionsForm
     validates :permanent_city, presence: true, if: -> { confirmed_permanent_address == "no" }
     validates :permanent_zip, presence: true, if: -> { confirmed_permanent_address == "no" }
 
+    def initialize(intake = nil, params = nil)
+      if params[:confirmed_permanent_address] == "yes"
+        [:permanent_apartment, :permanent_street, :permanent_city, :permanent_zip].each do |param|
+          params[param] = ""
+        end
+      end
```

Comment:
i bet we do something like this (conditionally remove/modify params from the form) elsewhere but i couldn't find an example

Reply:
either way let me know if this is good, neutral, or janky

Reply:
this looks good, but it might be helpful to document this with a unit spec